testing/upload_validation.ts

```diff
@@ -1069,7 +1069,7 @@ function validateFormulas(schema: z.ZodObject<any>) {
           return true;
         }
 
-        const codaDomains = ['coda.io', 'codahosted.io', 'localhost', 'calc-grpc-proxy'];
```

Comment:
Ok I remember a bit more now. It's not really about the fact that `codahosted.io` needs to use an auth token, it's that we don't want to generate an error if you include that. If you use an `Image` or `File` param type, it'll come through as a url on codahosted.io that you can download. You're free to declare codahosted.io as a network domain alongside coda.io and it's not supposed to count as an error, those two together are just what defines "connecting to Coda".

Now, I think we made it so that you don't have to declare codahosted.io at all anymore so it's just part of the system, so these days you don't even need to declare it. But if you do it shouldn't be an error.

And then I think `localhost` and `calc-grpc-proxy` are declared alongside `coda.io` by the Coda Debug pack which I guess needs to use Coda API token auth, and I think this carveout was just so that pack doesn't get an error during upload. I'm fine however we want to handle the carveout but we should avoid breaking that pack.

Reply:
Seems like removing `codahosted.io` should be ok here since only authenticated domains are being checked now. You could still declare a `codahosted.io` network as long as it's not an authenticated network

The coda debug pack is already updated to only use `coda.io` as an authenticated domain: https://github.com/coda/packs/blob/main/packs/coda/debug/manifest.ts